### PR TITLE
[VIS] Resources graph update, about page added

### PR DESCRIPTION
**Pull request by augusta100**
# Summary

Critical threshold level and common pool resources added to resources graph. About page added. 

Screenshot 2021-01-08 at 10 34 58
Screenshot 2021-01-08 at 10 35 12

### CODE REVIEW

File: website/src/components/About/About.tsx [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+const About = () => {
+  return (
+    <div style={{ paddingTop: 24 }}>
```
Comment (yp717) [inline website/src/components/About/About.tsx]: can you add more padding on the top here? something like paddingTop: 50px I think would work better

File: website/src/components/Visualisations/Resources/LineGraph.tsx [hunk 2]
```diff
@@ -44,15 +47,16 @@ class LineRechartComponent extends React.Component<IProps, any> {
...
         team5: 'ClientInfos.Team5.Resources',
         team6: 'ClientInfos.Team6.Resources',
+        CommonPool: 'CommonPool',
       },
       chartData: this.props.output,
     }
   }
 
-  handleClick = (dataKey) => {
+  handleClick = (dataKey: any) => {
```
Comment (Darrekt) [inline website/src/components/Visualisations/Resources/LineGraph.tsx]: why was this necessary?

File: website/src/components/Visualisations/Resources/LineGraph.tsx [hunk 3]
```diff
@@ -44,15 +47,16 @@ class LineRechartComponent extends React.Component<IProps, any> {
...
     this.setState({
       disabled: this.state.disabled.includes(dataKey)
-        ? this.state.disabled.filter((obj) => obj !== dataKey)
+        ? this.state.disabled.filter((obj: any) => obj !== dataKey)
```
Comment (Darrekt) [inline website/src/components/Visualisations/Resources/LineGraph.tsx]: same here, in general the use of `any` is frowned upon and can lead to a lot of runtime errors later on.

**Review by yp717 — CHANGES_REQUESTED** [review]
aside from the top spacing lgtm - there are some any types that could maybe be typed a bit more strictly but @Darrekt will address those :)

**Review by Darrekt — CHANGES_REQUESTED** [review]
Looks good but I would like to help you address the `any` bits. Let me know if you want to sit down about it together.

File: website/src/components/Visualisations/Resources/LineGraph.tsx (hunk 2 above)
Comment (augusta100) [reply website/src/components/Visualisations/Resources/LineGraph.tsx]: It was throwing an error and that was the quick fix, changed it to string now, is that okay?

File: website/src/components/Visualisations/Resources/LineGraph.tsx (hunk 3 above)
Comment (augusta100) [reply website/src/components/Visualisations/Resources/LineGraph.tsx]: Same with this one, changed to string

**Review by yp717 — APPROVED** [review]
lgtm

File: website/src/components/Visualisations/Resources/LineGraph.tsx [hunk 4]
```diff
@@ -16,26 +16,54 @@ import {
...
   ReferenceLine,
+  ReferenceArea,
+  TooltipProps,
 } from 'recharts'
 import _ from 'lodash'
 import { OutputJSONType } from '../../../consts/types'
+import styles from './Resources.module.css'
+
+const CustomTooltip = (props: TooltipProps) => {
+  const { active, label, payload } = props
```
Comment (Darrekt) [inline website/src/components/Visualisations/Resources/LineGraph.tsx]:
```suggestion
const CustomTooltip = ({ active, label, payload }: TooltipProps) => {
```

**Review by Darrekt — APPROVED** [review]
